src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/GenerateTemporaryTargetAssembly.cs

```diff
@@ -516,8 +516,8 @@ public string BaseIntermediateOutputPath
         /// Set this property to 'false' to use the .NET Core 3.0 behavior for this task. 
         ///
         /// </summary>
-        public string IncludePackageReferencesDuringMarkupCompilation 
-        { get; set; }
+        public bool IncludePackageReferencesDuringMarkupCompilation
+        { get; set; } = true;
```

Comment:
I believe that the value being sent to `set` is a string, and I'm not sure if it's okay to store that in a bool variable. As far as I know, string values are not automatically typecasted to bool.